packages/clutter_gtk.rb

```diff
@@ -4,7 +4,7 @@ class Clutter_gtk < Package
   description 'The Clutter Gtk package is a library providing facilities to integrate Clutter into GTK+ applications.'
   homepage 'https://wiki.gnome.org/Projects/Clutter'
   compatibility 'all'
-  @_app = name.tr('_', '-')
+  @_app = File.basename(__FILE__, '.rb').tr('_', '-')
```

Comment:
Does this work?
```
@_app = name.to_s.tr('_', '-')
```

Reply:
Probably no, as the `name` variable is initialized after `set_package` part (the `name` variable only works on `self.*` part)